Update changelog and docs for the new platforms (#324)
osohq/oso: docs/changelogs/vNEXT.rst

New features
==============

Windows Support
===============

> The three oso libraries (python, ruby and java) all now work on Windows.

Reviewer: ```suggestion
The three oso libraries (Python, Ruby and Java) all now work on Windows.
```

Musl Python Builds
===========

> Python linux musl-based wheels (for alpine and other musl based distros) are built and downloadable from the release in Github.

Reviewer: Might be nice to add a link here to the releases page.

> Musl Python Builds
> ===========
> Python linux musl-based wheels (for alpine and other musl based distros) are built and downloadable from the release in Github.

Reviewer 2: ```suggestion
musl builds for Python
======================

musl-based Python wheels (for Alpine Linux and other musl-based distros) are built and downloadable from [the releases page on GitHub](https://github.com/osohq/oso/releases).
```

Reviewer 2: ah crap, that's a markdown-style link

Reviewer 3: We're going to publish it as `oso-alpine` right?

Reviewer 3: Or `oso-musl` I forget.

Reviewer 2: updated (I think)

Author: It's not published to pypi at all yet. It's just available as a download on the Github release.

New features
==============

Windows Support
===============
The three oso libraries (Python, Ruby and Java) all now work on Windows.

musl builds for Python
======================

> musl-based Python wheels (for Alpine Linux and other musl-based distros) are built and downloadable from `the releases page on GitHub <https://github.com/osohq/oso/releases>`_.

Reviewer 2: ```suggestion
musl-based Python wheels (for Alpine Linux and other musl-based distros) are built and downloadable from `the releases page on GitHub <https://github.com/osohq/oso/releases/latest>`_.
```
